site/docs/v1/tech/themes/_theme-troubleshooting.adoc

```diff
@@ -20,4 +20,4 @@ Anytime a request is made to a themed page and we are unable to identify the ten
 * Any themed pages such as /password/forgot when a `client_id` is not provided.
 * Edge case error conditions where FusionAuth doesn't have context to determine the application or tenant.
 
-If you see the default theme unexpectedly, ensure sure you are passing required parameters to the page so that it can determine the applicable application or tenant.
+If you see the default theme unexpectedly, ensure you are passing required parameters to the page so that it can determine the applicable application or tenant.
```

Comment:
Minor nitpick: I would spell things out a little more like "...so that it can determine the application or tenant and apply the correct theme." or something like that.